test/OpenTelemetry.ResourceDetectors.Azure.Tests/AzureResourceDetectorTests.cs

```diff
@@ -95,11 +95,44 @@ public void TestAzureVmResourceDetector()
         }
     }
 
+    [Fact]
+    public void AzureContainerAppsResourceDetectorReturnsResourceWithAttributes()
+    {
+        try
+        {
+            foreach (var kvp in AzureContainerAppsResourceDetector.AzureContainerResourceAttributes)
+            {
+                Environment.SetEnvironmentVariable(kvp.Value, kvp.Key);
+            }
+
+            // Special case for service.name and resource uri attribute
```

Comment:
nit: note about resource uri is typo?

Reply:
copy-paste error, this comment is not needed.